### PR TITLE
Skip interceptor for methods that are not registered in the server

### PR DESCRIPTION
**Pull request by anuraaga**

### CODE REVIEW

File: tests/test_server.py [hunk 1]
```diff
@@ -177,15 +177,19 @@ def test_aborting_interceptor(aio):
...
+@pytest.mark.parametrize("aio", [False, True])
+def test_method_not_found(aio):
     """Calling undefined endpoints should return Unimplemented.
 
     Interceptors are not invoked when the RPC call is not handled.
     """
-    intr = CountingInterceptor()
+    intr_type = AsyncCountingInterceptor if aio else CountingInterceptor
+    intr = intr_type()
     interceptors = [intr]
```
Comment (metasyn) [inline tests/test_server.py:188]: Could this be simplified to:

```python
interceptors = [AsyncCountingInterceptor() if aio else CountiningInterceptor()]
```

**Comment by d5h** [comment]
Thanks for fixing this @anuraaga ! It’s much appreciated!

One more small thing, could you [bump the version](https://github.com/d5h-foss/grpc-interceptor/blob/master/CONTRIBUTING.md#making-a-pull-request) please? It’s needed for a new release.

File: tests/test_server.py (hunk 1 above)
Comment (d5h) [reply tests/test_server.py:188]: It’s following the pattern used elsewhere in the tests. I agree it’s weird looking though and I forget why it’s like that. There’s something to be said for both simplifying it and for keeping it consistent. I’m fine either way so will leave it up to you.

Comment (anuraaga) [reply tests/test_server.py:188]: I have gone ahead and left it as is to be consistent with the other tests. My understanding is that while not many, some of the interceptors accept a parameter in their constructor so this pattern allows passing parameters without repetition. Figured may as well leave them as is, and just stick with it throughout.

**Review by anuraaga — COMMENTED** [review]
Thanks @d5h! sorry for missing the point in the contributor docs

**Pull request merged**